amplicon-pooling.ipynb

```diff
@@ -874,7 +874,9 @@
     "         'BarcodesAreRC': 'False',\n",
     "         'ForwardAdapter': '',\n",
     "         'ReverseAdapter': '',\n",
-    "         'HumanFiltering': 'True'\n",
+    "         'HumanFiltering': 'True',\n",
+    "         'library_construction_protocol': '',\n",
```

Comment:
```suggestion
    "         'library_construction_protocol': 'Illumina EMP protocol 515fbc, 806r amplification of 16S rRNA V4',\n",
```

Reply:
that default amplicon language sounds good. Will we have to input that long string every time, or will it be the default value?

Reply:
Our current thought is that the default will be None (and the notebook will output an error since nothing has been input), but that we will have suggested values such as the current language in the comments. Does that sound reasonable?

Reply:
Sounds reasonable. You should maybe touch base with MacKenzie and Gail for the language of the suggested values.

Reply:
I think for the amplicon notebook and the library_construction_protocol we'll leave that there as the default value. For the experiment_design_description we want the default to be None so that an appropriate value is filled in.